docs/en/Architecture-DataFlow.md

```diff
@@ -67,14 +65,24 @@ with other components; it only responds to requests via RPC services.
 The leading master records all file system transactions to a distributed persistent storage
 to allow for recovery of master state information; the set of records is referred to as the journal.
 
-#### Standby Masters
+### Standby Masters
+
+Standby masters are launched on different servers to provide fault-tolerance when running Alluxio
```

Comment:
To improve the grammar here I would probably say "the Alluxio master in HA mode", "an Alluxio master in HA mode", or "when running Alluxio in HA mode"

Reply:
fixed